.github/scripts/update-changelog.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# set -o errexit # Abort if any command fails
+me=$(basename "$0")
+help_message="\
+Usage:
+  $me <version>
+
+Arguments:
+  version   The version of the current release."
+
+version=$1
+
+if [ -z "$version" ]; then
+    echo "Missing required argument <version>."
+    echo "$help_message"
+    exit 1
+fi
+
+echo "Updating changelog for release ${version}"
+
+release_notes=`cat RELEASE-NOTES.md`
+changelog=`sed "s/^# Changelog//" CHANGELOG.md`
+
+begin_notes="<!--- Begin Release ${version} -->"
+end_notes="<!--- End Release ${version} -->"
+
+if grep -F "Begin Release ${version}" CHANGELOG.md
+then
+    echo "Rewriting release ${version} notes."
+
+    new_changelog=`sed '1,/'"${end_notes}"'/d' CHANGELOG.md`
```

Comment:
Codacy found an issue: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/gh/SwedbankPay/design.swedbankpay.com/pullRequest?prid=5870223)